app/api/jobs/routine/[id]/executions/route.ts

```diff
@@ -0,0 +1,72 @@
+import { NextRequest, NextResponse } from "next/server";
+import { getDb } from "@/db/connection";
+import { RoutineJobsTable } from "@/db/schema";
+import { eq } from "drizzle-orm";
+import { VellumClient } from "vellum-ai";
+
+export async function GET(
+  request: NextRequest,
+  { params }: { params: Promise<{ id: string }> }
+) {
+  try {
+    const { id } = await params;
+    
+    const db = getDb();
+    const routineJob = await db
+      .select({ name: RoutineJobsTable.name })
+      .from(RoutineJobsTable)
+      .where(eq(RoutineJobsTable.id, id))
+      .limit(1);
+
+    if (routineJob.length === 0) {
+      return NextResponse.json(
+        { error: "Routine job not found" },
+        { status: 404 }
+      );
+    }
+
+    const workflowDeploymentName = routineJob[0].name;
+
+    const apiKey = process.env.VELLUM_API_KEY;
+    if (!apiKey) {
+      return NextResponse.json(
+        { error: "VELLUM_API_KEY not configured" },
+        { status: 500 }
+      );
+    }
+
+    const vellumClient = new VellumClient({ apiKey });
+
+    const deployments = await vellumClient.workflowDeployments.list();
+    const deployment = deployments.results?.find(
+      (d) => d.name === workflowDeploymentName
+    );
+
+    if (!deployment) {
+      return NextResponse.json(
+        { error: "Workflow deployment not found" },
+        { status: 404 }
+      );
+    }
+
+    const executions = await vellumClient.workflowDeployments.listExecutions(
+      deployment.id,
+      { limit: 10 }
+    );
```

Comment:
this is the wrong api, vercel is failing here:
<img width="995" height="223" alt="Screenshot 2025-08-12 at 9 27 59 PM" src="https://github.com/user-attachments/assets/b1dc673c-6d0c-4220-b9ce-5563b1ca8b18" />

You should use this method instead: https://docs.vellum.ai/developers/client-sdk/workflows/deployments/list-executions